### DISCUSSION TITLE
Different cams saving clips to different folders

### DISCUSSION THREAD

**Discussion by crider17** [blakeblackshear/frigate#682] [Ask A Question]
Can I have my cams save to different folders like so:

under cam 1 settings
snapshots:
  enabled: true
clips:
  enabled: true
save_clips:
  clips_dir: /media/cams/front

under cam 2 settings
snapshots:
  enabled: true
clips:
  enabled: true
save_clips:
  clips_dir: /media/cams/back

When I do this, in the log I get:
Error parsing config: extra keys not allowed @ data['cameras']['back']['save_clips']

Thanks!

**Comment by blakeblackshear** [comment]
No. The directory structure is not customizable because changing it will break functionality in Homeassistant. If you want to change the file names and locations, you will need to use your own scripts to do so.
